x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/detector_description.tsx

```diff
@@ -63,49 +58,82 @@ function createDetectorDescription(jobCreator: RareJobCreator, detectorType: RAR
   const populationFieldName = jobCreator.populationField?.id;
   const splitFieldName = jobCreator.splitField?.id;
 
-  const beginningSummary = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummary',
+  const rareSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSummary',
     {
-      defaultMessage: 'detects rare values of {rareFieldName}',
+      defaultMessage: 'Detects rare {rareFieldName} values.',
       values: { rareFieldName },
     }
   );
 
-  const beginningSummaryFreq = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummaryFreq',
+  const rareSplitSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSplitSummary',
     {
-      defaultMessage: 'detects frequently rare values of {rareFieldName}',
-      values: { rareFieldName },
+      defaultMessage: 'For each {splitFieldName} value, detects rare {rareFieldName} values.',
```

Comment:
Not sure about the first use of `value` here. Do you think it's needed?
e.g.
```For each host.name, detects rare process.name values.```
compared to 
```For each host.name value, detects rare process.name values.```

Reply:
Thanks! I've made that change in 19fdfd5678461ec4ac2bcaa18ea718a51bdec201